### DISCUSSION TITLE
redirecting different roles to different page on login

### DISCUSSION THREAD

**Discussion by bhagya1101** [directus/directus#17295] [Archive]
is it possible that Admin user should be on some page when they login and Non-Admin Users should be on some other page .
I dont want users to go on their `last_access` page.

Secondly, when I login by SAML, I see `origin` as undefined but login is successful?

**Comment by paescuj** [comment]
> is it possible that Admin user should be on some page when they login and Non-Admin Users should be on some other page .
I dont want users to go on their last_access page.

Not entirely sure if this would work, but you could do something similar to this: https://github.com/directus/directus/discussions/16095#discussioncomment-3930543
